### ISSUE TITLE
Introduction_李鸿利

### ISSUE THREAD

**Issue by williamlihongli** [hbwhzk/a11y-Programming-Guide-for-iOS#8]
张老师，你修改之后我看了介绍部分觉得有以下几个问题，请您参考：

1。 Using iOS 3.0 and later, VoiceOver is available to help users with visual impairments use their iOS-based devices. The UI Accessibility programming interface, introduced in iOS 3.0, helps developers make their applications accessible to VoiceOver users. Briefly, VoiceOver describes an application’s user interface and helps users navigate through the application’s views and controls, using speech and sound. Users familiar with VoiceOver in Mac OS X can leverage their experience to help them quickly come up to speed using VoiceOver on their devices.
iOS 3.0及之后的版本，视觉障碍用户可通过VoiceOver使用iOS设备。iOS 3.0引入的UI无障碍编程接口，帮助开发人员构建能够让VoiceOver用户访问的应该。简单地说，VoiceOver描述一个应用的用户界面，用户通过语音朗读以及声音，导航浏览该应用中的视图及控件。熟练掌握Mac OS X的VoiceOver用户，能够快速上手，利用其设备上的VoiceOver。
改：
iOS 3.0及之后的版本，视觉障碍用户可通过VoiceOver使用iOS设备。iOS 3.0引入的UI无障碍编程接口，帮助开发人员构建能够让VoiceOver用户访问的应用。简单地说，VoiceOver描述一个应用的用户界面，用户通过语音朗读以及声音，导航浏览该应用中的视图及控件。熟练掌握Mac OS X的VoiceOver用户，能够快速上手，利用其设备上的VoiceOver。
说明：
1）、 “帮助开发人员构建能够让VoiceOver用户访问的应该”改成“帮助开发人员构建能够让VoiceOver用户访问的应用”
**已改**
2）、“熟练掌握Mac OS X的VoiceOver用户，能够快速上手，利用其设备上的VoiceOver。”改成“熟练掌握Mac OS X的VoiceOver的用户，能够利用其设备上的VoiceOver快速上手。”感觉更加适合中文用户
**我想，快速上手是动词形态的形容词，后面应该跟名词。**
2、 iPhone applications that run in iOS 3.0 and later should be accessible to VoiceOver users. iOS and the iOS SDK support this goal by:
运行在iOS 3.0及之后版本的iPhone应用，应该为VoiceOver用户构建无障碍环境。iOS和iOS SDK通过以下方式支持此目标的:
改：
运行在iOS 3.0及之后版本的iPhone应用，应该为VoiceOver用户构建无障碍环境。iOS和iOS SDK通过以下方式支持此目标:
说明：“把最后的”的“删除更通顺
**已改**
3、Providing tools that help you implement accessibility in your code and test the accessibility of your application
提供相关工具，在代码中实施、并在应用中测试无障碍特性
改：
提供相关工具，在代码中实施无障碍、并在应用中测试无障碍特性
说明：感觉在“实施”之后加上“无障碍”更通顺，原文里面也是这样的。
**accessibility翻译成无障碍其实一直是很别扭的译法，最好的译法是可访问性，因为access是访问的意思，ibility是能力的意思，整体来说，这个词在目前状态下，翻译成无障碍特性，也是没办法，但是整句贯穿在一起就特别啰嗦。**
4、Understanding Accessibility on iOS briefly describes how VoiceOver works on the device and introduces the programming interface and tools you can use to make your application accessible.
“理解iOS上的无障碍特性”简要描述VoiceOver如何在设备上的运行，同时介绍了可用于构建应用可访问性的编程接口及工具。
改：
“理解iOS上的无障碍特性”简要描述VoiceOver如何在设备上运行，同时介绍了可用于构建应用可访问性的编程接口及工具。

说明：“简要描述VoiceOver如何在设备上的运行”把这里的“的”删除感觉更加通顺
**已改**
